src/NuGet.Licenses/Views/Shared/_Layout.cshtml

```diff
@@ -1,6 +1,6 @@
 ﻿<!DOCTYPE html>
 
-<html>
+<html lang="en">
```

Comment:
What consequences will it have on pages with non-English content? Like the following, for example: https://www.nuget.org/packages/%E6%97%A5%E6%9C%AC%E8%AA%9E%E3%82%B5%E3%83%B3%E3%83%97%E3%83%AB%E3%83%87%E3%83%BC%E3%82%BF/